### ISSUE TITLE
question/feature request: user history?

### ISSUE THREAD

**Issue by annamorphism** [DDMAL/CantusDB#1129]
On the admin pages for OldCantus, it was possible to see a list of users ordered by most recent log-in/changes made. I think I can see this information on individual user pages, but not in a list--is that true? If is it true, can we make there be a way to see something like this:

<img width="1217" alt="image" src="https://github.com/DDMAL/CantusDB/assets/67451875/bcb0e335-8cbb-4f47-83f9-ee202b27b27a">

It's not often useful, but for questions like "did this user change anything when they encountered Bug X" it would be nice to have.

**Comment by jacobdgm** [comment]
I don't believe we have this page. This (the proliferation of user pages on OldCantus) was on my list of things to ask Debra, but I don't recall that we got a clear directive about this.
